### PR TITLE
add Lifi widget to get sDAI with Popover (only desktop for now)

### PR DESCRIPTION
**Pull request by Diogomartf**
Makes it easy for people to get sDAI token and bet on presagio.

We are out of space on the top navbar on mobile, it's out of scope of this PR to fix that. 
This can work with `popover` out of the box and feels nicer than modal on desktop.

**screenshots with Popover**
image

**screenshots modal (previous)**
image
image

### CODE REVIEW

File: app/(main)/layout.tsx [hunk 1]
```diff
@@ -1,4 +1,6 @@
 import { Footer, Navbar } from '@/app/components';
+import { LifiWidget } from '@/app/components/LifiWidget';
```
[the remark this answers is not in the document]
Comment (Diogomartf) [reply app/(main)/layout.tsx]: fixed.

File: app/(main)/layout.tsx [hunk 2]
```diff
@@ -14,6 +15,7 @@ export default function MainLayout({
           <Navbar />
           {children}
           <Footer />
+          <LifiWidgetModal />
```
Comment (berteotti) [inline app/(main)/layout.tsx:18]: Why are we declaring this here? Why not inside Navbar? I think we should declare where it's being used to be easier to find and understand.

Comment (ElRodrigote) [reply app/(main)/layout.tsx:18]: Like the settings popover? I like this suggestion

Comment (Diogomartf) [reply app/(main)/layout.tsx:18]: Yeah, it's better

**Review by ElRodrigote — APPROVED** [review]

File: app/components/LifiWidgetPopover.tsx [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  return (
+    <Popover>
+      <PopoverTrigger asChild>
+        <Button variant="pastel" className="space-x-2 text-nowrap" onClick={() => {}}>
+          <p>Get {SDAI.symbol}</p>
+          <TokenLogo address={SDAI.address} size="xs" className="hidden md:block" />
+        </Button>
+      </PopoverTrigger>
+      <PopoverContent className="max-w-80 px-4 md:max-w-lg">
+        <LifiWidget />
```
Comment (berteotti) [inline app/components/LifiWidgetPopover.tsx:18]: Should we delete this component and have the logic here? I think in this case there is not so much code, so it's good to have everything in 1 file.

Comment (Diogomartf) [reply app/components/LifiWidgetPopover.tsx:18]: yeah, that's a good idea.

**Review by berteotti — APPROVED** [review]

File: app/components/LifiWidgetPopover.tsx [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+  return (
+    <Popover>
+      <PopoverTrigger asChild>
+        <Button variant="pastel" className="space-x-2 text-nowrap">
+          <p>Get {SDAI.symbol}</p>
+        </Button>
+      </PopoverTrigger>
+      <PopoverContent className="max-w-80 px-4 md:max-w-lg">
+        <LiFiWidget integrator="Presagio" config={widgetConfig} />
```
Comment (ElRodrigote) [inline app/components/LifiWidgetPopover.tsx:42]:
```suggestion
        <LiFiWidget integrator={APP_NAME} config={widgetConfig} />
```

**Review by ElRodrigote — APPROVED** [review]

File: app/components/LifiWidgetPopover.tsx (hunk 4 above)
Comment (Diogomartf) [reply app/components/LifiWidgetPopover.tsx:42]: yes

**Pull request merged**